tests/lib/files.sh

```diff
@@ -45,8 +45,8 @@ clean_dir() {
 ensure_file_exists() {
```

Comment:
The name of the function is surprising because nothing mentions the extra `.fake` file being created.

Reply:
Yeah, I can look at this in a followup, I inherited this one.